napari_cellseg3d/code_models/model_instance_seg.py

```diff
@@ -50,18 +109,48 @@ def get_dict(self):
 
 
 def threshold(volume, thresh):
+    """Remove all values smaller than the specified threshold in the volume"""
     im = np.squeeze(volume)
     binary = im > thresh
     return np.where(binary, im, np.zeros_like(im))
 
 
+def voronoi_otsu(
```

Comment:
Might be helpful for some people to include a link to the code for the Voronoi Otsu method in the doc string:
https://github.com/clEsperanto/pyclesperanto_prototype/blob/master/pyclesperanto_prototype/_tier9/_voronoi_otsu_labeling.py

Reply:
Solved in #36